obs-packaging/linux-container/debian.rules

```diff
@@ -3,14 +3,21 @@ include /usr/share/dpkg/default.mk
 KernelDir=debian/kata-linux-container/usr/share/kata-containers
 DebugDir=debian/kata-linux-container-debug/usr/share/kata-containers
 KernelVer=${DEB_VERSION_UPSTREAM_REVISION}.container
+arch=$(shell arch)
+KATA_MULTIARCH="/usr/src/packages/SOURCES/kata-multiarch.sh"
+KERNEL_ARCH=$(shell /bin/bash $(KATA_MULTIARCH) -a $(arch))
+KERNEL_IMAGE=$(shell /bin/bash $(KATA_MULTIARCH) -i $(arch))
+
 %:
 	dh $@
 
 override_dh_auto_build:
 	sed -i "s/^EXTRAVERSION.*/EXTRAVERSION = -$$(echo $(KernelVer) | cut -d'-' -f2)/" Makefile
-	cp /usr/src/packages/SOURCES/config .config
-	make -s ARCH=x86_64 oldconfig > /dev/null
-	make -s CONFIG_DEBUG_SECTION_MISMATCH=y ARCH=x86_64 -j4 all
+	tar xzf /usr/src/packages/SOURCES/kata-configs.tar.gz
+	find kata-configs -name "$(KERNEL_ARCH)_kata_kvm_*" -exec cp {} .config \;
```

Comment:
Not related to this PR, but this reminds me, it would be useful if we packaged the `.config` files (gzip'd) along with the kernel images themselves.

Reply:
Configs are included in the `-debug` package, but I agree that's something that may be needed also for non-debug purposes.